src/pybind/tests/test_kaldi_pybind.py

```diff
@@ -28,9 +28,9 @@ def test_float_vector(self):
         gold = np.array([0, 0, 2, 2, 2])
         self.assertTrue((np_array==gold).all())
 
-    def test_float_matrxi(self):
+    def test_float_matrix(self):
         # test FloatMatrix
-        print("=====Testing FloatVector in kaldi_pybind=====")
+        print("=====Testing FloatMatrix in kaldi_pybind=====")
```

Comment:
this file is not properly formatted.
For instance, the following statement

```python
kp_matrix = kp.FloatMatrix(4,5)
```

There should be a space after the comma, i.e., `4, 5`.

After fixing the typos, could you also fix the format ?

Reply:
sure. done.